epan/dissectors/packet-ssh.c

```diff
@@ -539,6 +544,24 @@ parse_ssh_file(const gchar *file_path, guchar **key0, guint *len0, guchar **key1
     parse_ssh_key(val_1, key1, len1);
 }
 
+typedef struct {
+  guint32 seqnr;
+  gboolean is_sent;
+} ssh_packet_private_data_t;
+
+static ssh_packet_private_data_t*
+ssh_get_packet_data(packet_info *pinfo, gboolean is_sent, guint32 current_seq)
+{
+  ssh_packet_private_data_t *packet_data = (ssh_packet_private_data_t*) p_get_proto_data(pinfo->pool, pinfo, proto_ssh, 0);
+  if (!packet_data) {
+    packet_data = wmem_new0(pinfo->pool, ssh_packet_private_data_t);
```

Comment:
You probably want to use `wmem_file_scope()` here instead of `pinfo->pool`. The former remains present while the capture file is open while the latter is invalidated when the current packet changes.

Reply:
Done!